doc/usermanual/CMakeLists.txt

```diff
@@ -281,9 +283,137 @@ if(can_build_usermanual)
 			install(FILES ${pdffile} DESTINATION ./share/doc/darktable/usermanual OPTIONAL)
 		endforeach(language)
 	endif()
+#
+# LUA API GENERATION
+#
+
+	#
+	# draft pdf API
+	#
+	add_custom_command(
+		OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/darktable-api.pdf
```

Comment:
`darktable-lua-api.pdf` might be better, once day we might add a C API, or finally document the internal one ...